core/modules/user/js/user.admin.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file
+ * Attaches behaviors for the User module.
+ */
+(function ($) {
+
+Backdrop.behaviors.userFieldsetSummaries = {
+  // Provide the vertical tab summaries.
+  attach: function (context) {
+    var $context = $(context);
+
+    // Account settings.
+    $context.find('fieldset#edit-account-settings').backdropSetSummary(function() {
+      var vals = [];
+
+      // Status.
+      var status = $context.find('input[name="status"]:checked').parent().find('label').text();
+      vals.push($.trim(status));
+
+      // Roles.
+      var roles = [];
+      var checkedRoles = $context.find('input[name^="roles"]:not([disabled]):checked');
+      checkedRoles.each(function() {
+        roles.push($.trim($(this).parent().find('label').text()));
+      });
+      if (roles.length) {
+        var rolesText = Backdrop.t('Roles:') + ' ' + roles.join(', ');
+      }
+      else {
+        var rolesText = Backdrop.t('No roles');
+      }
+      vals.push(rolesText);
+
+      return Backdrop.checkPlain(vals.join(', '));
+    });
+
+    // Personalization.
+    $context.find('fieldset#edit-personalization').backdropSetSummary(function() {
+      var vals = [];
+
+      // Signature.
+      var signature = $context.find('textarea[name="signature[value]"]');
```

Comment:
Extremely nitpicky and unnecessary comment follows, and core is quite variable with this, but few other examples in this file where `find()` returns a JQuery object, and convention suggests we name the variable with the `$`